src/apiCalls.js

```diff
@@ -34,6 +34,25 @@ const handleRecipeData = recipes => {
 
 const handleIngredientData = ingredients => pageData.allIngredients = ingredients
 
+const patchHits = (recipe) => {
+  fetch('http://localhost:3001/api/v1/recipeHits', {
+    method: 'PATCH',
+    body: JSON.stringify({recipeID: recipe.id}),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+    .then(res => res.json())
+    .then(data => {
+      if(!data.message.includes('Success')) {alert(data.message)}
```

Comment:
do we really want to alert if this fails?

Reply:
Do we want to just console.error the message instead? What do you suggest we change that to

Reply:
Since this patch is for the admin and not user, i feel it should not be displayed at all whether the patch is successful or not? The user would not do anything wrong to impact a failed patch, so I feel they should not get feedback for the same.